src/Support/Field.php

```diff
@@ -94,7 +99,38 @@ public function getValidator(array $args, array $rules): ValidatorContract
         return Validator::make($args, $rules, $messages);
     }
 
-    protected function getResolver(): ?Closure
+    protected function getResolver() : ?Closure
+    {
+        $resolver = $this->originalResolver();
+
+        if (! $resolver) {
+            return null;
+        }
+
+        return function ($root, ...$arguments) use ($resolver) {
+            return app(Pipeline::class)
+                ->send(array_merge([$this], $arguments))
+                ->through($this->middleware)
+                ->via('resolve')
+                ->then(function ($arguments) use ($resolver, $root) {
+                    $result = $resolver($root, ...array_slice($arguments, 1));
+
+                    foreach ($this->middleware as $name) {
+                        $middleware = App::make($name);
```

Comment:
Why are we using `app()` above and `App::make` here? (feels inconsistent)

I guess using the Facade would be preferred? (aka death to global functions)

Reply:
Yah I just copied this out of Laravel and never thought about it, facade it is.